lib/motors/controller.hpp

```diff
@@ -1,10 +1,19 @@
 #pragma once
+#include <cstdint>
+
+#include "core/logger.hpp"
 
 namespace hyped::motors {
+enum class controller_status { ControllerTemperatureExceeded, GeneralWarning, Nominal };
```

Comment:
ControllerStatus
variables are lowercase underscores, classes and enums are upper camel case

Reply:
Probably call it `kUrecoverableWarning`, or similar to make it clear what it is